creator-node/scripts/run-tests.sh

```diff
@@ -40,3 +40,7 @@ mkdir -p $storagePath
 ./node_modules/mocha/bin/mocha --timeout 30000 --exit
 
 rm -r $storagePath
+
```

Comment:
do we need to add the delegate wallet and private key here for local testing?

Reply:
yes, i've been manually adding these values to the default-config.json for local testing, but adding here would be much more effective. will add keys here

Reply:
perfect! i was actually just about to comment on the resize image test and ask you to move your delegate key pair into the run-tests.sh file. if you remove it from default-config.json and add it to run-tests.sh and make sure it's consistent in both pr's i think we can merge these!